### ISSUE TITLE
Training with larger image size 

### ISSUE THREAD

**Issue by abhigoku10** [bubbliiiing/yolov4-pytorch#240]
@bubbliiiing  thanks for your effort and sharing the open-source code i have few queries for my requirement 

Req: I am re-training the yolov4 on COCO dataset for 1056x1056 input image resolution 
I have changed only the image size rest of the parameters remains the same by default 
1. The training process is taking a long time, for two days only 20 epoch is completed? what is the ideal time taking for the epochs
2. For 20 epoch i am getting detections are very less ie in an image of 6 ppl only 2 ppl are getting detected, so should we allow the training for a longer duration what is the optimum time to see promising results 
3. for 20 epochs the loss is around~ 10 when can we see the drop in loss values 
4. Should we change any other hyper params when training for higher resolution 

Please share your thoughts and thanks in adavance

**Comment by bubbliiiing** [comment]
sorry

1. Your input picture is a little big. If such a big picture is enhanced online, the data set processing process will be very long, which is not conducive to training. Maybe the sizes of 416 and 640 will be more normal.
2. 20 epochs are certainly not enough in theory. You see, yolox is 300 epochs.
3. You can look at the loss of the model loaded with Coco's pre training weight, which is a reasonable value
4. I don't have any ideas about high-resolution pictures. I still have a lot to learn about the selection of super parameters. If you have corresponding materials, please

**Comment by abhigoku10** [comment]
@bubbliiiing  thanks for your response 
Currently training is going well thanks for the code 
1. What is the difference between darknet repo and your repo in terms of code and tricks 
2. how much time does it take to train a model from scratch on 2V100 systems for 640 img resolution 
3. Currently the model is trained with coco dataset can i add crowd human, city persons dataset along with coco and see an increase in accuracy of person detection? 
Please share your thoughts

**Comment by bubbliiiing** [comment]
1. I reproduced some of his skills. Due to the limitation of computing power, I can't start training coco from 0. Generally speaking, the model will start training coco from the backbone trained by Imagenet. If it starts training completely from zero, the effect will not be particularly good. Unless there is a better data enhancement method, you can know from yolox's paper. There are some parameter details in the Darknet library, which I haven't looked at in detail.

2. The speed is limited to the data loading speed. I'll consider whether to add a piece of code so that users can load all pictures into the network and reduce the data loading time

3. Theoretically, the increase of data set can increase the prediction accuracy, but it depends on the test set.
